.github/CONTRIBUTING.md

```diff
@@ -18,14 +18,14 @@ root of this project to launch CouchDB. Below are the steps:
 
 1. Start [Docker](https://docs.docker.com/get-docker/). Install it if you don't have it yet.
 2. Install [Docker Compose](https://docs.docker.com/compose/install/) if you don't have it yet.
-3. Run `docker-compose up --build -d` in the root directory.
+3. Run `./couchdb/couchdb-init.sh` in the root directory.
```

Comment:
We need to add that this works only on Unix based systems. On Windows, we still need to let users type `docker-compose up --build -d` directely.

Reply:
You are right, I will add this.